packages/app-config/src/config/appConfigBase.ts

```diff
@@ -18,9 +18,9 @@ const apis: AppConfig["apis"] = {
     gateway: "https://ipfs.filebase.io/ipfs/",
     pinning: "https://api.filebase.io/v1/ipfs",
     buckets: {
-      nfts: "cfce-give-nfts",
-      avatars: "cfce-profiles",
-      media: "cfce-media",
+      nfts: "kuyawa-public", // TODO: revert back to cfce-give-nfts
```

Comment:
revert back plz

Reply:
btw, you can always create a custom appConfig (appConfig.kuyawa.ts) with your own values and use a custom APP_ENV

Reply:
Good to know